spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -298,7 +444,20 @@ the first two reads utilize keys, while the third is executed using a query base
 
 ===== Indexes
 
-This feature is not supported yet.
+By default, all fields are indexed. To disable indexing on a particular field, `@Unindexed` annotation can be used.
+
+Example:
+[source, java]
+----
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Unindexed;
+
+public class ExapleItem {
+	private long indexedField;
```

Comment:
not a big deal, but this example class doesn't have getters and setters, so it looks strange that its properties are both "private"